src/java.base/share/classes/java/lang/StringLatin1.java

```diff
@@ -79,6 +117,130 @@ public static byte[] inflate(byte[] value, int off, int len) {
         return ret;
     }
 
```

Comment:
I think this is an improvement since it better groups these latin1 formatting methods together. Add a copy of the comment from `StringUTF16` about bound-checking here?